documentation/API_documentation/DeviceLocation_API.md

```diff
@@ -0,0 +1,99 @@
+| **HTTP Request**<br> POST \<base-url>/location/v0/verify<br>**QueryParameters**<br> No query parameters are defined.<br>**Path Parameters**<br> No path parameters are defined.<br>**Request Body Parameters**<br> **ueId**: UE identifier object, contains 4 different identifiers, at least 1 has to be set<br> **port (optional):** User equipment port. Device port may be required along with IP address to identify the target device <br> **latitude:** Latitude component of the location.<br> **longitude:** Longitude component of the location<br> **accuracy:** Accuracy (radius) of the location check in km <br><br>**Response**<br> **200: Location verified**<br>  Response body:<br>   **verificationResult:** bolean result value <br><br> **400:** **Invalid input.**<br> **401:** **Un-authorized, missing or incorrect authentication.**<br> **405:** **Invalid input**<br> **500:** **Session not created**<br> **503:** **Service temporarily unavailable** |
+<br>
+
+### 4.3 Errors
+
+Since CAMARA Location API is based on REST design principles and blueprints, well defined HTTP status
+codes and families specified by community are followed [2].
+
+Details of HTTP based error/exception codes for the Location API are described in Section 4.2 of each API REST based method.
+Following table provides an overview of common error names, codes and messages applicable to Location API.
+
+| No  | Error Name          | Error Code | Error Message                                                 |
+|-----|---------------------|------------|---------------------------------------------------------------|
+| 1   | Invalid port(s)     | 400        | "Ports specification not valid                                |
+| 2   | Invalid ueId        | 400        | "Validation failed for parameter: ueId"                       |
+| 3   | Invalid port        | 400        | "Validation failed for parameter: port"                       |
+| 4   | Invalid latitude    | 400        | "Validation failed for parameter: latitude"                   |
+| 5   | Invalid longitude   | 400        | "Validation failed for parameter: longitude"                  |
+| 6   | Invalid accuracy    | 400        | "Validation failed for parameter: accuracy"                   |
+| 7   | Unauthorized        | 401        | "Un-authorized to invoke operation"                           |
+| 8   | Forbidden           | 403        | "Forbidden to invoke operation"                               |
+| 9   | Service unavailable | 503        | “Internal error due to required telco service unavailability" |
+
```

Comment:
Some errors are missing, see above.
Actually the service should return in error case, beside the http response code an errorcode + errormessage

see yaml file: 

example:
            code: INVALID_INPUT
            message: 'Invalid input'

Unfortunately the implementation doesn't seem to be correct either...